internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```diff
@@ -296,4 +296,56 @@ AutoScalingPolicy{{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}Ev
 {{- end }}{{/* range $topic := .Subscribe.Topics */}}
 {{- end }}{{/* if .Subscribe */}}
 
-{{- end }}{{/* if .Autoscaling.QueueDelay */}}
\ No newline at end of file
+{{- end }}{{/* if .Autoscaling.QueueDelay */}}
+
+{{- if .Autoscaling.Requests}}
```

Comment:
How about for worker service?

Reply:
I wonder if we need something like this to make our validations more specific:

```go
type backendTaskConfigValidator TaskConfig
type workerTaskConfigValidator TaskConfig

type backendCountValidator Count 
type workerCountValidator Count

type (v backendCountValidator) Validate() error {
  if !v.QueueScaling().IsEmpty() {
      return errors.New("boom")
  }
  ...
}

if (v workerCountValidator) Validate() error {
   if v.Requests != nil {
       return errors.New("boom2")
   }
}
```

and to use it:
```go
func (b BackendServiceConfig) Validate() error {
  ...
  if err := backendTaskConfigValidator(b.TaskConfig).Validate(); err != nil { ... }
  ...
}
```

I understand that it's a massive change though, and I'm okay with addressing it as a follow-up PR.

Today, the same problem I think exists for LBWS and BackendServices if users specify `queue_delay` for example.

Reply:
Chatted offline - we'll wait to make this refactor in a follow up PR.